src/app/notifications/notifications-page/notifications-page.component.html

```
@@ -30,7 +30,8 @@
         mat-row
         *matRowDef="let row; columns: displayedColumns"
         class="select-row"
-        [routerLink]="[routeMap[row.objectType], row.objectId]"
+        (click)="navigateToNotification(row)"
+        style="cursor: pointer"
```

HTMLHint (1.8.0)

[warning] Inline style [ 
        style="cursor: pointer" ] cannot be used.

(inline-style-disabled)